auto_dev/commands/run.py

```diff
@@ -28,9 +28,34 @@
 
 
 TENDERMINT_RESET_TIMEOUT = 10
-TENDERMINT_RESET_ENDPOINT = "http://localhost:8080/hard_reset"
 TENDERMINT_RESET_RETRIES = 20
 
+def _verify_flask_port(container: Any, port: int) -> bool:
+    """Verify Flask is running on the given port."""
+    health_check = container.exec_run(f"curl -s http://127.0.0.1:{port}/")
+    return health_check.exit_code == 0
+
+def _get_flask_port(container_name: str = "tm_0") -> int:
+    """Get the Flask port from the container."""
+    client = docker.from_env()
+    try:
+        container = client.containers.get(container_name)
+        # Wait up to 10 seconds for port file to exist and contain a port
+        for _ in range(10):
+            result = container.exec_run("cat /tmp/port.txt")
+            if result.exit_code == 0:
+                # Get last line that contains only digits
+                for line in reversed(result.output.decode().splitlines()):
+                    if line.strip().isdigit():
+                        port = int(line.strip())
+                        if _verify_flask_port(container, port):
+                            return port
+            time.sleep(1)
+        return 8080  # fallback to default
+    except (docker.errors.NotFound, docker.errors.APIError, ValueError) as e:
+        print(f"Error getting Flask port: {e}")
+        return 8080  # fallback to default
+
```

Comment:
i think you may need to do a subfunction here, i think that is why lint is failing

Reply:
changed it up to appease the linter in de039ad7d1d3a8989cb6ff342b06895aefda9012